src/codec_avm.c

```diff
@@ -451,46 +381,6 @@ static avifBool avifProcessAOMOptionsPostInit(avifCodec * codec, avifBool alpha)
         if (!strcmp(key, "tune")) {
             codec->internal->tuningSet = AVIF_TRUE;
         }
-
-        avifBool match = AVIF_FALSE;
-        for (int j = 0; aomOptionDefs[j].name; ++j) {
-            if (avifKeyEqualsName(entry->key, aomOptionDefs[j].name, alpha)) {
-                match = AVIF_TRUE;
-                avifBool success = AVIF_FALSE;
-                int valInt;
-                unsigned int valUInt;
-                switch (aomOptionDefs[j].type) {
-                    case AVIF_AOM_OPTION_NUL:
-                        success = AVIF_FALSE;
-                        break;
-                    case AVIF_AOM_OPTION_STR:
-                        success = aom_codec_control(&codec->internal->encoder, aomOptionDefs[j].controlId, entry->value) == AOM_CODEC_OK;
-                        break;
-                    case AVIF_AOM_OPTION_INT:
-                        success = aomOptionParseInt(entry->value, &valInt) &&
-                                  aom_codec_control(&codec->internal->encoder, aomOptionDefs[j].controlId, valInt) == AOM_CODEC_OK;
-                        break;
-                    case AVIF_AOM_OPTION_UINT:
-                        success = aomOptionParseUInt(entry->value, &valUInt) &&
-                                  aom_codec_control(&codec->internal->encoder, aomOptionDefs[j].controlId, valUInt) == AOM_CODEC_OK;
-                        break;
-                    case AVIF_AOM_OPTION_ENUM:
-                        success = aomOptionParseEnum(entry->value, aomOptionDefs[j].enums, &valInt) &&
-                                  aom_codec_control(&codec->internal->encoder, aomOptionDefs[j].controlId, valInt) == AOM_CODEC_OK;
-                        break;
-                }
-                if (!success) {
-                    return AVIF_FALSE;
-                }
-                if (aomOptionDefs[j].controlId == AOME_SET_TUNING) {
-                    codec->internal->tuningSet = AVIF_TRUE;
-                }
-                break;
-            }
-        }
-        if (!match) {
-            return AVIF_FALSE;
-        }
```

Comment:
Removing this block of code is a bug fix.

Reply:
I just tried this and it works. So we can land this and I can delete #2714

Reply:
Frank: Thanks for verifying this works. I have landed this.